morphic.js

```diff
@@ -9403,16 +9439,34 @@ MenuItemMorph.prototype.mouseEnter = function () {
         this.image = this.highlightImage;
         this.changed();
     }
+    if (this.doubleClickAction instanceof MenuMorph) {
+        var w = this.world();
```

Comment:
please declare all local variables in a single ```var``` .statement at the top of the scope, i.e. the function, not inside code blocks that don't have their own scope. Also, ```w``` could be spelled out ```world``` :)